packages/css/src/components/buttons.css

```diff
@@ -1,138 +1,140 @@
-    color: var(--button-color);
-    height: var(--button-height);
-    box-shadow: inset 0 0 0 2px var(--button-border-color);
-    padding: 0 var(--button-side-padding);
-    display: inline-flex;
-    position: relative;
-    justify-content: center;
-    align-items: center;
-    text-align: center;
-    overflow: hidden;
-    box-sizing: border-box;
-    border-width: 0;
-    text-decoration: none;
-  }
-  .vtmn-btn:disabled {
-    opacity: 0.25;
-    cursor: not-allowed;
-  }
-  .vtmn-btn:not([disabled]):hover {
-    cursor: pointer;
-    background-color: var(--button-background-color-hover);
-  }
-  .vtmn-btn:not([disabled]):active {
-    background-color: var(--button-background-color-active);
+    --btn-border-color: transparent;
+
+    box-shadow: inset 0 0 0 0.125rem var(--btn-border-color);
+    min-width: calc(6 * 1rem);
+    letter-spacing: 0.0375rem;
+
+    @apply vtmn-font-display
+      vtmn-bg-brand-digital
+      vtmn-text-white
+      vtmn-text-base
+      vtmn-font-bold
+      vtmn-py-0
+      vtmn-px-5
+      vtmn-relative
+      vtmn-inline-flex
+      vtmn-no-underline
+      vtmn-rounded
+      vtmn-h-8
+      vtmn-border-0
+      vtmn-box-border
+      vtmn-overflow-hidden
+      vtmn-text-center
+      vtmn-items-center
+      vtmn-justify-center
+
+      disabled:vtmn-opacity-25
+      disabled:vtmn-cursor-not-allowed
+
+      hover:vtmn-cursor-pointer
+      hover:vtmn-bg-brand-digital-dark-1
+
+      active:vtmn-bg-brand-digital-dark-2;
   }
+
   .vtmn-btn:not([disabled]):focus {
     outline: none;
-    box-shadow: inset 0 0 0 2px var(--button-border-color),
+    box-shadow: inset 0 0 0 2px var(--btn-border-color),
       0 0 0 2px var(--vtmn-color_brand-digital-light-1);
   }
 
   /* Primary button */
   .vtmn-btn_variant--primary {
-    --button-background-color: var(--vtmn-color_brand-digital);
-    --button-background-color-hover: var(--vtmn-color_brand-digital-dark-1);
-    --button-background-color-active: var(--vtmn-color_brand-digital-dark-2);
-    --button-background-color-focus: var(--vtmn-color_brand-digital);
-    --button-color: var(--vtmn-color_white);
-    --button-border-color: transparent;
+    --btn-border-color: transparent;
+
+    @apply vtmn-bg-brand-digital
+      vtmn-text-white
+      hover:vtmn-bg-brand-digital-dark-1
+      active:vtmn-bg-brand-digital-dark-2;
   }
 
   /* Primary reversed button */
   .vtmn-btn_variant--primary-reversed {
-    --button-background-color: var(--vtmn-color_white);
-    --button-background-color-hover: rgba(255, 255, 255, 0.8);
-    --button-background-color-active: rgba(255, 255, 255, 0.6);
-    --button-background-color-focus: var(--vtmn-color_white);
-    --button-color: var(--vtmn-color_black);
-    --button-border-color: transparent;
+    --btn-border-color: transparent;
+
+    @apply vtmn-bg-white
+      vtmn-text-black;
+  }
+  .vtmn-btn_variant--primary-reversed:not([disabled]):hover {
+    background-color: rgba(255, 255, 255, 0.8);
```

Comment:
why not use SASS (with darken or lighten) ?

Reply:
Thanks @amen-souissi, done with a PostCSS function here: https://github.com/Decathlon/vitamin-web/pull/37/commits/d02cd37ac64fdb780a591d64c6b51f8f12a71f45